__tests__/app.test.js

```diff
@@ -121,46 +137,99 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 
-  describe("Get /api/articles", () => {
-    test("Get ALL ARTICLES sorted by created by", () => {
+  describe("Get /api/articles/:article_id/comments", () => {
+    test("Get comments by article id", () => {
+      const testArticleId = 1;
       return request(app)
-        .get("/api/articles/")
+        .get(`/api/articles/${testArticleId}/comments`)
         .then((response) => {
-          const retrievedArticles = response.body;
-          expect(response.status).toBe(200);
-          expect(new Date(retrievedArticles[0].created_at)).toEqual(
-            new Date("2020-11-03T09:12:00.000Z")
-          );
-          for (let i = 1; i < retrievedArticles.length; i++) {
-            expect(
-              new Date(retrievedArticles[i].created_at).getTime()
-            ).toBeLessThanOrEqual(
-              new Date(retrievedArticles[i - 1].created_at).getTime()
-            );
-          }
+          expect(200);
+          expect(Array.isArray(response.body)).toBe(true);
+          response.body.forEach((comment) => {
+            expect(comment).toMatchObject({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              article_id: testArticleId,
+            });
+          });
         });
     });
-    test("Articles should have a comment count property", () => {
+    test("Comments should be sorted by newest first based on the created_at property", () => {
+      const testArticleId = 6;
       return request(app)
-        .get("/api/articles/")
+        .get(`/api/articles/${testArticleId}/comments`)
         .then((response) => {
-          const retrievedArticles = response.body;
-          retrievedArticles.forEach((article) => {
-            expect(article).toHaveProperty("comment_count");
-          });
+          const retrievedComments = response.body;
+
+          expect(200);
+          for (let i = 1; i < retrievedComments.length; i++) {
+            expect(
+              retrievedComments[i].created_at.toBeLessThanOrEqual(
+                retrievedComments[i - 1].created_at
+              )
+            );
+          }
```

Comment:
One extra happy path test here for a valid article with no associated comments status 200 returns an empty array